students/jarosz_natalia/lesson_08/regex_version_of_strip.py

```diff
@@ -0,0 +1,14 @@
+import re
+
+
+def regex_strip(string, chars_to_remove=' '):
+    strip_regex = re.compile("^([" + chars_to_remove + "]*)")
```

Comment:
Almost ok. You handle here the only case when ```chars_to_remove``` is at the beginning of the string. Now you should do a similar thing for the case when ```chars_to_remove``` is at the end.
Your line 5 should look like:
```strip_regex = re.compile("^([" + chars_to_remove+ "]*) | ([" + chars_to_remove+ "]*)$")```